src/python/run_matching.py

```diff
@@ -97,6 +97,11 @@ def main():
     args = parseParams()
     yaml_config = setDictParam(args)
 
+    if (args.output_dir.exists()):
+        print("WARNING: output_dir exists. Overwritting the results")
```

Comment:
nit: you seem to be using the `LOG(WARNING)` in other places, so why not also here?

Reply:
Yeah, for this I need to add glogging to Python too.
Will get to it when polishing the final script for end2end processing with images.